docs/index.rst

```diff
@@ -1,20 +1,19 @@
 :mod:`pyramid_nacl_session`
 ===========================
 
-This Pyramid add-on defines an encrypting, pickle-based cookie serializer,
+This Pyramid add-on defines an encrypting, pickle-based cookie serializer, 
```

Comment:
Space at the end of a line is likely to get stripped by the next person to edit the file (I have it set up to do it automatically on files since in most cases it's a mistake), it is likely better to add this to the beginning of the next line.